### ISSUE TITLE
Please add Smalltalk support

### ISSUE THREAD

**Issue by Kirtai** [o2sh/onefetch#343]
When used in a Smalltalk repo like https://github.com/Cuis-Smalltalk/Cuis-Smalltalk-Dev it misidentifies it as CSS.

**Comment by HallerPatrick** [comment]
What even is the Smalltalk logo?

Is it this?

![Smalltalk-powered](https://user-images.githubusercontent.com/22773355/101990576-eb4f9b80-3ca7-11eb-9dcc-89224a519ee7.gif)

So should the ascii art be a hot air balloon?

**Comment by Kirtai** [comment]
Yes, it's a colourful, striped hot air balloon.  Searching for "Smalltalk balloon" will give lots of examples. The original was on the cover of Byte Magazine (https://archive.org/details/byte-magazine-1981-08)[here].

**Comment by spenserblack** [comment]
Currently, onefetch uses tokei to detect languages. It appears that tokei doesn't yet support smalltalk (see XAMPPRocky/tokei#316).

If you're interested in adding Smalltalk support to tokei, it's very easy -- all you should need to do is add a new object with a few keys to a JSON file.

**Comment by spenserblack** [comment]
Linking #1152 to PRs and issues that will probably be affected.
